01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,64 @@
+const { writeFile, readFile } = require("fs").promises;
+
+async function writer() {
+  try {
+    await writeFile(
+      "./writtenFile.txt",
+      "Hi there!\n",
+      {
+        encoding: "utf8",
+        flag: "w",
+      },
+      () => {
+        try {
+          console.log("Creating ------>  File created");
+        } catch (error) {
+          console.log("Error: " + error);
+        }
+      }
+    );
+    await writeFile(
+      "./writtenFile.txt",
+      "Second line\n",
+      {
+        encoding: "utf8",
+        flag: "a",
+      },
+      () => {
+        try {
```

Comment:
These `try-catch` statements can be removed (the ones that only have `console.log` statements inside)